core/src/main/scala/chisel3/IO.scala

```diff
@@ -94,13 +95,26 @@ object FlatIO {
 
     type R = T with Record
     gen match {
+      case d if chisel3.reflect.DataMirror.hasProbeTypeModifier(d) => IO(d)
```

Comment:
```suggestion
      case d if hasProbeTypeModifier(d) => IO(d)
```
Please just `import chisel3.reflect.DataMirror.hasProbeTypeModifier`

Reply:
done